backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/codec/impl/ElasticsearchBigDecimalFieldCodec.java

```diff
@@ -65,6 +65,11 @@ public BigDecimal decode(JsonElement element) {
 		return JsonElementTypes.BIG_DECIMAL.fromElement( element );
 	}
 
+	@Override
+	public BigDecimal convertFromDoubleNullSafe(Double value) {
+		return BigDecimal.valueOf( value );
```

Comment:
It's probably best to move these converter methods to the utils similar to `ParseUtils`.